Assets/_Scripts/Systems/LevelSystem.cs

```diff
@@ -33,14 +34,20 @@ public enum LevelState
 
         public void Start()
         {
-            NetworkSystem.Instance.OnSceneLoadDoneEvent += ChangeLevelState;
             networkSceneObjectProvider = gameObject.AddComponent<NetworkSceneMaganer>();
         }
 
+        // Since the NetworkRunner is deleted after a connection error (idk why),
```

Comment:
Est-ce qu'il faut le mettre dans un try catch?

Reply:
Mm, mettre quoi? La création du network system?

Reply:
N'importe quel call dans le Network system qui pourrait crash?

Reply:
Ah yeah, j'ai essayé au début du concours de catch l'exception mais sans succès. Je sais pas si tu te souviens, je vous avais montré le phénomène. C'est sûr que je pourrais toujours investiguer davantage afin d'éviter cela. À voir si on a le temps.

Reply:
xD j'avais oublié. Go ahead squash and merge this bad boy